src/config.rs

```diff
@@ -351,6 +416,12 @@ pub struct IpAddressStatic {
     pub address: IpAddr,
 }
 
+#[derive(Clone, PartialEq, Debug, Deserialize)]
+pub struct IpAddressStun {
+    pub stun_server: String,
+    pub ipv6: bool
```

Comment:
It is a bit implicit that `ipv4` should be used if `ipv6` is set to false. I think it would be clearer to use an enum here instead of a boolean.

Maybe
```
pub enum AddressType {
  IPV4, IPV6
}
```